cli/popper/gha.py

```diff
@@ -226,12 +236,13 @@ class ActionRunner(object):
     """An action runner.
     """
 
-    def __init__(self, action, workspace, env, timeout, verbose):
+    def __init__(self, action, workspace, env, timeout, verbose, dry_run):
         self.action = action
         self.workspace = workspace
         self.env = env
         self.timeout = timeout
```

Comment:
@ivotron during conflict resolution I came across `self.timeout`. Has it been removed? It is not there in master?https://github.com/systemslab/popper/blob/ab14a6a2b554cf673fa90880ad78be7ae1211281/cli/popper/gha.py#L280-L285

Reply:
yeah, sorry about that. We're removing that in favor of `ctrl-c`

Reply:
@ivotron so many merge conflicts :'(
Is my assumption right that `verbose` has been renamed to `debug`?